src/software/world/game_state.h

```diff
@@ -390,6 +390,14 @@ class GameState
      */
     bool operator!=(const GameState& other) const;
 
+    /**
+     * Returns the point on the field where the ball should be placed if one is specified.
+     *
+     * @return the requested ball placement position on the field if one is specified,
+     * otherwise std::nullptr
```

Comment:
```suggestion
     * otherwise std::nullopt
```
use `std::nullopt` for `std::optional`

Reply:
fixed